samples/apps/copilot-chat-app/webapp/src/components/chat/chat-list/ChatList.tsx

```diff
@@ -55,22 +66,74 @@ const useClasses = makeStyles({
             display: 'none',
         }),
     },
+    input: {
+        ...shorthands.padding(tokens.spacingHorizontalNone),
+        ...shorthands.border(tokens.borderRadiusNone),
+        width: 'calc(100% - 24px)',
+        backgroundColor: 'transparent',
+        fontSize: '20px',
+    },
 });
 
 export const ChatList: FC = () => {
     const classes = useClasses();
-    const { conversations, selectedId } = useAppSelector((state: RootState) => state.conversations);
+    const chat = useChat();
+
+    const [isFiltering, setIsFiltering] = useState(false);
```

Comment:
Is `isFiltering== true` when `searchString !== ""` ? Do we need two local states?

Reply:
Yeah, filtering is a different UI state, so the search string can be empty when the User first clicks the Filter icon. In this case, all conversations are shown
![image](https://github.com/microsoft/semantic-kernel/assets/125500434/3e373ad0-23be-4842-8d93-d3a37e5e6786)

But I've removed searchString from state. I realized that doesn't need to be a controlled variable since we're controlling the filtering state, so the value will be controlled by input component mount / unmount

Reply:
https://github.com/microsoft/semantic-kernel/pull/1248#pullrequestreview-1446913040